src/frontend/src/lib/components/core/Menu.svelte

```diff
@@ -68,6 +75,17 @@
 
 		<Hr />
 
-		<SignOut on:icLogoutTriggered={() => (visible = false)} />
+		<AboutWhat asMenuItem on:icOpenAboutModal={hidePopover} />
+		<AboutHow asMenuItem on:icOpenAboutModal={hidePopover} />
+
+		<Hr />
+
+		<SignOut on:icLogoutTriggered={hidePopover} />
 	</div>
 </Popover>
+
+{#if $modalAboutWhat}
```

Comment:
That does not work I think, aren't the modal duplicated in the UI when you open it? Like two About modals one above another one?

Reply:
these are inside the Menu component that is rendered once logged in, while the others are inside the AboutMenu that is rendered only when logged out. Is that what you meant?

Reply:
do you prefer to put them both outside and in the `Header` component?

Reply:
Right, it works for now indeed. Nevertheless, that's prone to duplication, I think it would be worth declaring the modals only once. Maybe we can move them one level higher?

Reply:
perfect! moving them to `Header`